CMakeLists.txt

```diff
@@ -234,6 +237,25 @@ if(USE_GAMEMATH)
     endif()
 endif()
 
+if(USE_SDL2)
```

Comment:
I'd like to see an attempt to find SDL2 first and fall back to building SDL2 ourselves if we don't find it with a warning message informing that SDL2 wasn't found and that we are going to build it ourselves.